lib/sqlitex/server.ex

```diff
@@ -43,6 +48,10 @@ defmodule Sqlitex.Server do
     GenServer.call(pid, {:query, sql, opts})
   end
 
+  def create_table(pid, name, cols) do
+    GenServer.call(pid, {:create_table, name, cols})
```

Comment:
The `Sqlitex.create_table` function takes table options.  Should this server function take the same optional table options?